src/sdk/ConversationTranslator/Conversation.ts

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// Multi-device Conversation is a Preview feature.
+
+import { IDisposable } from "../../common/Exports";
+import { Contracts } from "../Contracts";
+import { PropertyCollection, SpeechTranslationConfig } from "../Exports";
+import { IConversation } from "./IConversation";
+import { IParticipant, IUser } from "./IParticipant";
+
+export abstract class Conversation implements IConversation {
+
+    public abstract get authorizationToken(): string;
+    public abstract set authorizationToken(value: string);
+
+    public abstract get config(): SpeechTranslationConfig;
```

Comment:
Please remove these. Setting the SpeechConfig after a conversation has started is not a supported operation. Getting the SpeechConfig and then changing a property there also leads to an expectation that would change the Conversation which it does not.